types/sap__approuter/my-ext.ts

```diff
@@ -0,0 +1,17 @@
+module.exports = {
```

Comment:
Whats the purpose of this file? Please move this to tests instead

Reply:
it's for testing, moved it to tests folder